### PR TITLE
Header touch ups

### PR DESCRIPTION
**Pull request by djnunez-aot**
List of header changes:
All buttons should have a height of 40px

Reduce height of the image to 56px and reduce the margin on the top and bottom of the image to 16px so total height of the header is reduced to about 88px.

Change the color of the bottom border to #d8d8d8, RGB 216, 216, 216

Add a separator between the logo and EPIC.submit

For @jadmsaadaot :
Change Epic.Submit to EPIC.submit for VITE_APP_TITLE  in env on Openshift

### CODE REVIEW

File: submit-web/src/components/Shared/layout/Header/EAOAppBar.tsx [hunk 1]
```diff
@@ -14,29 +14,31 @@ export default function EAOAppBar() {
         position="static"
         color="inherit"
         sx={{
-          borderBottom: `1px solid ${theme.palette.grey[300]}`,
+          borderBottom: `1px solid #d8d8d8`,
```
Comment (jadmnsaad) [inline submit-web/src/components/Shared/layout/Header/EAOAppBar.tsx]: how can we mitigate using a custom color like this, can we explore using a color that exists in BCDesignTokens?

File: submit-web/src/styles/theme.tsx [hunk 2]
```diff
@@ -1,6 +1,16 @@
...
 
 // Any theme overrides should be passed into the createAppTheme.
 
-export const theme = createAppTheme({});
+export const theme = createAppTheme({
+  components: {
+    MuiButton: {
+      styleOverrides: {
+        root: {
+          height: "40px",
```
Comment (jadmnsaad) [inline submit-web/src/styles/theme.tsx]: I'm worried about this, this has caused problems in Track where text has been squished and sometimes it overflowed especially when the text wraps. Maybe also then not allow text to wrap in buttons (some css property)

Comment (djnunez-aot) [reply submit-web/src/styles/theme.tsx]: how should we set this up then?

File: submit-web/src/components/Shared/layout/Header/EAOAppBar.tsx (hunk 1 above)
Comment (djnunez-aot) [reply submit-web/src/components/Shared/layout/Header/EAOAppBar.tsx]: i looked through the bcdesign tokens i did not find d8d8d8

Comment (jadmnsaad) [reply submit-web/src/components/Shared/layout/Header/EAOAppBar.tsx]: then this should prompt a conversation. Let's talk with MA about only using BCDesignTokens

File: submit-web/src/styles/theme.tsx (hunk 2 above)
Comment (jadmnsaad) [reply submit-web/src/styles/theme.tsx]: please look into it or remove this and do it in another PR

**Review by jadmnsaad — APPROVED** [review]

**Pull request merged**